documentation/quickstart-qemu.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started with Qemu Worker
+
+This is a quick start guide for using the leviathan framework with the qemu worker.
+
+## Start your first test run
+
+For your first test run, we will be running the [e2e test suite](https://github.com/balena-os/leviathan/tree/master/suites/e2e).
+
+A balenaOS image that is used to flash and provision the device under test will be needed for the test. You can download an unmanaged image from [balena.io/os](balena.io/os). Extract the downloaded balenaOS image you want to test with to the `workspace` directory and rename it to `balena.img` file.
+
+## Build your config.json file
+
+The config.js file is the master configuration file for your test run. The qemu worker spins up and configures your device under test (DUT) accordingly with the settings provided in the `config.js` file. To know more about each property, refer to the {@page Config.js Reference | Config.js reference}.
+
+To get started,
+
+- Navigate to the `workspace` directory in leviathan.
+- Create a `config.js` file in the `workspace` directory using the `config.example.js` file.
+- The config.js file should look close to the following:
+
+```js
+module.exports = {
+    deviceType: 'genericx86-64-ext',
+    // Multiple suites can be run on the same worker when specified as an array.
+    suite: `${__dirname}/../suites/e2e`,
+    config: {
+        networkWired: false,
+        networkWireless: false,
+        downloadVersion: 'latest',
+        balenaApiKey: process.env.BALENACLOUD_API_KEY,
+        balenaApiUrl: 'balena-cloud.com',
+        organization: 'BALENACLOUD_ORG_GOES_HERE',
+    },
+    image: `${__dirname}/balena.img`,
+    workers: ['http://localhost'],
+};
+```
+
+Here, we are assuming that the device under test (DUT) is a `raspberrypi3`. The suite and the image property points to a valid test suite and balenaOS image that is going to be tested with. 
+
+## Start your first test run
+
+To start the test run, navigate to the root of the leviathan directory and run the following command:
+
+```
+make local-test
+```
+
+This starts to build the core and worker services and run the worker using docker-compose. The logs will start streaming on the terminal for the test run.After then, the test will start. Wait for the test scenario to finish and check the device logs on the dashboard in the meantime. 
```

Comment:
```suggestion
This will first build the core and worker services and run the worker using docker-compose. Then, the test will run and the logs will start streaming on the terminal. Wait for the test scenario to finish and check the device logs on the dashboard in the meantime. 
```